src/tests/jobtest.c

```diff
@@ -30,26 +30,130 @@
 
 #include <stdio.h>
 #include <string.h>
+#include <strings.h>
 #include <daos.h>
+#include <daos_mgmt.h>
+#include <daos/common.h>
+#include <daos/pool.h>
+
+static char *progname;
+
+void print_usage()
+{
+	fprintf(stderr, "Usage: %s -p pool_str [-s nsecs] [-x] \
```

Comment:
(style)  Avoid line continuations in quoted strings

Reply:
Cant avoid this and keep it to less than 80 characters.